### DISCUSSION TITLE
Various Movements/speed UP/DOWN

### DISCUSSION THREAD

**Discussion by gessima** [sn4k3/UVtools#197] [Ideas]
Would it be possible to get a 2step Movementspeed - let's say on moving 8mm down - the first 6mm with 3mm/s and the last 2mm with 1mm/s that could be a big advantage for the needed time of a print.

**Comment by sn4k3** [comment]
I wish... all this printers with encoded formats are trash... really you are buying something you can't fully use. FDM printers are far, far, but far ahead mSLA in that field.

Only Gcode printers are capable of what you are asking.
See some of them here: https://github.com/sn4k3/UVtools/wiki/Printer-compability-with-per-layer-settings-and-advanced-tools , look at bottom printers with the [gcode] keyword
